application/forms/PreferenceForm.php

```diff
@@ -185,7 +185,8 @@ public function createElements(array $formData)
                 false
             );
         }
-
+        $themeFile = null;
+        
```

Comment:
We require empty lines after closed control structures if any instructions follow. Though, none when starting control structures, so if you flip both lines around I'm fine with it.